css/style.css

```diff
@@ -9,285 +15,2094 @@ body {
+  .new-сollection {
+    background-position: -100px 92px;
+  }
+}
+
+/* Медиа запросы 2й секции */
+
+@media (max-width: 1400px) {
+  .card {
+    width: 35%;
+    margin-left: 60%;
+  }
+}
+
+@media (max-width: 1400px) {
+  .bag__link {
+    width: 40%;
+    margin-left: 60%;
+  }
+}
+
+@media (max-width: 1400px) {
+  .bag__arrow {
+    margin-left: 78%;
+    width: 15%;
+  }
+}
+
+@media (max-width: 1400px) {
+  .bag__wight {
+    width: 27%;
+    margin-left: 5%;
+  }
+}
+
+@media (max-width: 1400px) {
+  .bag__pink {
+    width: 35%;
+    margin-left: 15%;
+    height: 750px;
+  }
+}
+
+@media (max-width: 1330px) {
+  .card {
+    width: 38%;
+  }
+}
+
+@media (max-width: 1300px) {
+  .bag__arrow {
+    margin-left: 80%;
+  }
+}
+
+@media (max-width: 1220px) {
+  .bag__title-bold {
+    font-size: 30px;
+    line-height: 31px;
+    left: 30px;
+    width: 300px;
+  }
+}
+
+@media (max-width: 1220px) {
+  .bag__title {
     font-size: 12px;
-    line-height: 15px;
+    line-height: 12px;
+    left: 30px;
+  }
 }
 
-.new-collection__background {
-    background-size: 100% auto;
-}
-
-
-    @media (max-width: 768px) { 
-        .logo,
-        .lang,
-        .check,
-        .new-сollection__social,
-        .text,
-        .new-сollection__main-card,
-        .new-collection__background {
-            display: none;
-            } 
-        } 
-
-        @media (min-width: 768px) { 
-        .logo-mobil,
-        .galka {
-            display: none;
-            } 
-        } 
-
-        @media (max-width: 768px) { 
-            .girl {
-                background-image: none;
-            }
-        }
- 
-        @media (max-width: 768px) { 
-            .new-сollection {
-               background: url(../img/girl2.jpg) no-repeat;
-               background-size: cover contain;      
-               background-position: 0 36%;        
-           }
-        }
-
-        @media (max-width: 768px) { 
-            .title {
-                font-size: 50px;
-                line-height: 51px;
-                position: absolute;
-                left: 5%;
-                top: 60%;
-                width: 297px;
-                height: 103px;
-           }
-        }
-
-        @media (max-width: 768px) { 
-            .galka {
-                position: absolute;
-                left: 170px;
-                top: 625px;
-           }
-        }
-       
\ No newline at end of file
+@media (max-width: 1220px) {
+  .bag-text {
+    width: 80%;
+    top: 150px;
+    left: 30px;
+  }
+}
+
+@media (max-width: 1220px) {
+  .bag__button-number {
+    width: 90px;
+    left: 30px;
+  }
+}
+
+@media (max-width: 1220px) {
+  .bag__color {
+    left: 30px;
+  }
+}
+
+@media (max-width: 1220px) {
+  .bag__button {
+    left: 140px;
+    width: 220px;
+  }
+}
+
+@media (max-width: 1220px) {
+  span {
+    left: 60%;
+  }
+}
+
+@media (max-width: 1220px) {
+  .card {
+    width: 35%;
+  }
+}
+
+@media (max-width: 1150px) {
+  .bag__arrow {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5449357)